### DISCUSSION TITLE
Xml documentation not showing in swagger UI

### DISCUSSION THREAD

**Discussion by tonven** [jasontaylordev/CleanArchitecture#996] [Q&A]
Hi. Was trying to add xml documentation visible in swagger. This is how comments look like on a property of my Query:
`/// <summary>
    /// Comma-separated list of Ids (e.g., 1,2,3)
    /// </summary>`
    
   Tried different solution, but still it does not work. Anyone managed to add documentation for properties in swagger?

**Comment by vantroysanchez** [comment]
Apply this configuration in the project properties:

![swagger xml](https://github.com/jasontaylordev/CleanArchitecture/assets/48096872/24358ff6-6080-45b1-9ab6-e92d570c7f7c)
